api/package.json

```diff
@@ -23,10 +23,14 @@
   "dependencies": {
     "@nestjs/common": "^7.0.0",
     "@nestjs/core": "^7.0.0",
+    "@nestjs/mongoose": "^7.0.2",
```

Comment:
use typeORM

Reply:
I still need more time to get very familiar with nestjs best practices.
rule #1 : it's working right now and we can refactor it.